apps/sms/test/marionette/new_activity_test.js

```diff
@@ -231,5 +231,31 @@ marionette('Messages as "new" activity target', function() {
         messagesApp.Composer.messageInput, 'Message input should be focused'
       );
     });
+
+    test('Activity with thread "number" and "body"', function() {
```

Comment:
Yay! I like integration tests, thanks for this one! Basically it makes me more confident to introduce risky changes when we have good coverage :)